### PR TITLE
Done with the Port to Python3 for this repo

### PR DESCRIPTION
**Pull request by kshitijdshah99**
@walterbender I have modified, port all the necessary file to python3 if any issues pls tell me.

### CODE REVIEW

File: classroombroadcast_activity.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# the Free Software Foundation; either version 2 of the License, or
```
Comment (chimosky) [inline classroombroadcast_activity.py:1]: This isn't a complete version of the license short version.

**Comment by chimosky** [comment]
Your commit messages are different from the actual changes themselves, and you've deleted files only to add them back, that's a bit confusing.

File: classroombroadcast_activity.py (hunk 1 above)
Comment (kshitijdshah99) [reply classroombroadcast_activity.py:1]: @chimosky I am not getting you completely....I used command line 2to3 . -w to port all necessary file to python 3 besides this I haven't made any modification

**Comment by kshitijdshah99** [comment]
Yeah really sorry for this actually there was some misunderstanding, hence made those wrong commits.

**Comment by quozl** [comment]
activity.info has not been changed; have you tested the activity after your change?

File: classroombroadcast_activity.py (hunk 1 above)
Comment (chimosky) [reply classroombroadcast_activity.py:1]: Look at other files in the activity and you'll see the difference between the shortened version of the License.

**Comment by kshitijshah99** [comment]
Yeah, I am working on it.

**Comment by quozl** [comment]
You are not the original poster.  I'm confused.

**Comment by kshitijdshah99** [comment]
Sorry, accidently I mailed here using my college email-id Github account.

File: classroombroadcast_activity.py (hunk 1 above)
Comment (kshitijdshah99) [reply classroombroadcast_activity.py:1]: @chimosky I have to replace the text you mentioned with the shortened license one given in [setup.py](https://github.com/sugarlabs/classroombroadcast/blob/master/setup.py), am I right?

**Comment by kshitijdshah99** [comment]
> activity.info has not been changed; have you tested the activity after your change?

I am unable to run the activity in sugar live build. I tried downloading the .XO file from Browse but am unable to start the classroombroadcast activity? Shall I raise an issue for this? It's looking something like this-
classroombroadcast error

**Comment by quozl** [comment]
Well, please fix the activity before making a pull request.  There's no point us reviewing a pull request until you have it working.  There are multiple issues already with this activity.  @tony37 raised some of them, but he died in December 2022.  But that doesn't invalidate what he found.  If the activity cannot be fixed, we should consider removing the repository.

**Pull request closed without merging**